component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/util/EmailOTPAuthErrorConstants.java

```diff
@@ -0,0 +1,51 @@
+package org.wso2.carbon.identity.authenticator.emailotp.util;
```

Comment:
License header and class comment missing.

Reply:
Added